commcare-cloud/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -377,21 +378,21 @@ def get_workers_to_work_on(self, args):
         """
         celery_config = self.get_celery_config(args)
         # full name of workers per host to run an action on
-        workers_by_host = defaultdict(list)
+        workers_by_host = defaultdict(set)
         if args.worker_name:
             for worker_name in args.worker_name.split(','):
                 if worker_name not in celery_config:
                     puts(colored.red(
                         "%s not found in the list of possible worker names, %s" % (
                             worker_name, celery_config.keys()
                         )))
-                    raise
+                    raise IncorrectOptions
```

Comment:
Not following that closely, but since the convention is to raise _instances_ of exception objects, I'd guess you want `IncorrectOptions()` here.

Reply:
Cool, this is now replaced with `assert`, thanks for flagging though..